media/List-QRB-Test-Bot.xml

```diff
@@ -0,0 +1 @@
+<?xml version="1.0"?><h:html xmlns="http://www.w3.org/2002/xforms" xmlns:ev="http://www.w3.org/2001/xml-events" xmlns:h="http://www.w3.org/1999/xhtml" xmlns:jr="http://openrosa.org/javarosa" xmlns:odk="http://www.opendatakit.org/xforms" xmlns:orx="http://openrosa.org/xforms" xmlns:xsd="http://www.w3.org/2001/XMLSchema"><h:head><h:title>List &amp; Button test v2</h:title><model odk:xforms-version="1.0.0"><instance><data id="List-Button-test-v2" version="1"><name/><third_choice/><third_list/><eof/><meta><instanceID/></meta></data></instance><bind nodeset="/data/name" type="string"/><bind nodeset="/data/third_choice" required="true()" stylingTags="buttonsForListItems" type="string"/><bind nodeset="/data/third_list" required="true()" stylingTags="list" type="string"/><bind nodeset="/data/eof" type="string"/><bind jr:preload="uid" nodeset="/data/meta/instanceID" readonly="true()" type="string"/></model></h:head><h:body><input ref="/data/name"><label>Welcome to test bot. What is your name?</label></input><select1 ref="/data/third_choice"><label>What is your gender?</label><item><label>1 Male</label><value>Male</value></item><item><label>2 Female</label><value>Female</value></item></select1><select1 ref="/data/third_list"><label>Select a skill from list?</label><item><label>1 c</label><value>c</value></item><item><label>2 photoshop</label><value>photoshop</value></item><item><label>3 java</label><value>java</value></item><item><label>4 python</label><value>python</value></item><item><label>5 html</label><value>html</value></item><item><label>6 css</label><value>css</value></item><item><label>7 java script</label><value>java script</value></item><item><label>8 react js</label><value>react js</value></item><item><label>9 node js</label><value>node js</value></item><item><label>10 angular js</label><value>angular js</value></item></select1><input ref="/data/eof"><label>Form Ended</label></input></h:body></h:html>
```

Comment:
Where are these 2 files getting used?

Reply:
I think these were added by pankak at some point. Since I did my changes on main and then raised a PR on dev, so these changes came along.